src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
Why have you added a trailing slash? Currently, it creates duplication, which is not good.
![image](https://user-images.githubusercontent.com/45945732/228861325-6a94a516-01e8-47f9-b7a9-ed36c603c9e1.png)